kudoctl/src/client/workload.rs

```diff
@@ -1,15 +1,32 @@
 use anyhow::{Context, Result};
-use log::debug;
+use log::{debug, warn};
 use reqwest::Method;
 use serde::{Deserialize, Serialize};
 
 use crate::{
     client::types::IdResponse,
-    resource::{self, workload},
+    resource::{
+        self,
+        workload::{self, Resources},
+    },
 };
 
 use super::request::{Client, RequestError};
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct PortBinding {
+    pub source: i32,
+    pub destination: i32,
+}
+#[derive(Debug, Deserialize, Serialize)]
+pub struct WorkloadDTO {
```

Comment:
What is a `WorkloadDTO` ?

Reply:
It’s the workload as stored in the controller, I added a comment in the code.